include/zephyr/bluetooth/audio/has.h

```diff
@@ -41,16 +41,23 @@ extern "C" {
 /** Preset name maximum length */
 #define BT_HAS_PRESET_NAME_MAX 40
 
-/** @brief Opaque Hearing Access Service object. */
-struct bt_has;
-
 /** Hearing Aid device type */
 enum bt_has_hearing_aid_type {
 	BT_HAS_HEARING_AID_TYPE_BINAURAL = 0x00,
 	BT_HAS_HEARING_AID_TYPE_MONAURAL = 0x01,
 	BT_HAS_HEARING_AID_TYPE_BANDED = 0x02,
 };
 
+/** Hearing Aid device features */
+enum bt_has_features {
+	BT_HAS_FEAT_HEARING_AID_TYPE_LSB = BIT(0),
+	BT_HAS_FEAT_HEARING_AID_TYPE_MSB = BIT(1),
+	BT_HAS_FEAT_PRESET_SYNC_SUPP = BIT(2),
+	BT_HAS_FEAT_INDEPENDENT_PRESETS = BIT(3),
+	BT_HAS_FEAT_DYNAMIC_PRESETS = BIT(4),
+	BT_HAS_FEAT_WRITABLE_PRESETS_SUPP = BIT(5),
+} __packed;
```

Comment:
Generally we don't pack `enum`s in the public API; I assume this is because you want to use it directly over air? 

Does packing an enum ensure the correct size?